src/fmgc/src/guidance/lnav/legs/VM.ts

```diff
@@ -0,0 +1,39 @@
+import { Degrees, NauticalMiles } from '@typings/types';
+import { ControlLaw, GuidanceParameters } from '@fmgc/guidance/ControlLaws';
+import { LatLongData } from '@typings/fs-base-ui/html_ui/JS/Types';
+import { Leg } from '@fmgc/guidance/lnav/legs';
+
+export class VMLeg implements Leg {
+    public heading: Degrees;
+
+    constructor(heading: Degrees) {
+        this.heading = heading;
+    }
+
+    get bearing(): Degrees {
+        return this.heading;
```

Comment:
Unfortunately, heading legs are not this simple. Here you set the bearing/course to the target heading, but that is not correct. Bearing should be the actual course of the leg. In general, the course is not the same as the heading, you would need to add the predicted wind to get the actual course. What is implemented here is the display of a CX leg, and the guidance of a VX leg.

Reply:
This is not meant to be an accurate first implementation. There is predicted leg wind at the moment for me to work with.

Reply:
This is not possible to correct at the moment.